src/libraries/System.Management/src/System/Management/ManagementScope.cs

```diff
@@ -1178,11 +1178,8 @@ internal int QueryObjectSink_(int lFlags, ref IWbemObjectSink ppResponseHandler)
         }
         internal int GetObject_(string strObjectPath, int lFlags, IWbemContext pCtx, ref IWbemClassObjectFreeThreaded ppObject, IntPtr ppCallResult)
         {
-            //It is assumed that caller always passes ppCallResult as IntPtr.Zero.
-            //If it changes let this call go through wminet_utils.dll. Check implementation of CreateInstanceEnum_ for more information.
             int status = (int)tag_WBEMSTATUS.WBEM_E_FAILED;
-            if (!object.ReferenceEquals(ppCallResult, IntPtr.Zero))
```

Comment:
try.dot.net and other experimentation tells me there's no unified boxing applying to IntPtr here.

The style after my edit looks weird, but it matches everything else in this class.